src/pybind/mgr/cephadm/services/nfs.py

```diff
@@ -70,6 +71,29 @@ def create(self, daemon_id, host, spec):
             daemon_id, host, spec))
         return self.mgr._create_daemon('nfs', daemon_id, host)
 
+    def config_dashboard(self, daemon_descrs: List[DaemonDescription]):
+        
+        def __get_set_cmd_dicts(out: str) -> List[dict]:
```

Comment:
```suggestion
        def get_set_cmd_dicts(out: str) -> List[dict]:
```
(strip the d-under?)

Reply:
Fixed. Thanks!